pygmt/src/rose.py

```diff
@@ -80,7 +80,7 @@ def rose(self, data=None, length=None, azimuth=None, **kwargs):
 
     orientation : bool
         Specifies that the input data are orientation data (i.e., have a
-        180 degree ambiguity) instead of true 0-360 degree directions
+        180 degrees ambiguity) instead of true 0-360 degree directions
```

Comment:
I'm not sure about this change. Google search returns "180 degree ambiguity", "180-degree ambiguity", and "180 degrees ambiguity".

Reply:
I changed this back in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/9053380919a4523010163e6331835ed46a2813f0. I also was unsure here (please see comment https://github.com/GenericMappingTools/pygmt/pull/2278#issuecomment-1365010563).